src/pages/docs/to-be-sorted/singleton/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Singleton"
+parent: "to-be-sorted"
```

Comment:
Meta will be built programatically, this will be overwritten by the forthcoming script to normalise all articles.